domains/ewan.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "personal portfolio for dev",
+  "repo": "https://github.com/Ewan-Dev/portfolio",
+  "owner": {
+    "username": "Ewan-Dev",
+    "email": "lyricizer2@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://ewan-dev.github.io"
```

```suggestion
    "CNAME": "ewan-dev.github.io"
```
CNAME cannot contain protocol (`https://`).